sdk/monitor/Azure.Monitor.Query/README.md

```diff
@@ -106,9 +109,11 @@ foreach (var logEntryModel in response.Value)
 If your query return a single column (or a single value) of a primitive type you can use `LogsClient.QueryAsync<T>` overload to deserialize it:
 
 ```C# Snippet:QueryLogsAsPrimitive
-LogsClient client = new LogsClient(TestEnvironment.LogsEndpoint, new DefaultAzureCredential());
+Uri endpoint = new Uri("https://api.loganalytics.io");
```

Comment:
Is this endpoint the same for all clouds?

Reply:
No, but I want to give the 95% user an easy starting point.